v_3_1_1/master_template.go

```diff
@@ -2402,8 +2402,9 @@ coreos:
       --pod-manifest-path=/etc/kubernetes/manifests \
       --kubeconfig=/etc/kubernetes/config/kubelet-kubeconfig.yml \
       --node-labels="node-role.kubernetes.io/master,role=master,kubernetes.io/hostname=${HOSTNAME},ip=${DEFAULT_IPV4},{{.Cluster.Kubernetes.Kubelet.Labels}}" \
-      --kube-reserved="cpu=150m,memory=250Mi" \
+      --kube-reserved="cpu=200m,memory=250Mi" \
       --system-reserved="cpu=150m,memory=250Mi" \
```

Comment:
Finally, I leave 200m for container runtime and kubelet after checking how other open tools (kops, rancher, openshift) set this value.